Tests/StreamChatTests/Workers/Background/AttachmentQueueUploader_Tests.swift

```diff
@@ -264,6 +264,52 @@ final class AttachmentQueueUploader_Tests: XCTestCase {
             Assert.willBeEqual(imageModel?.extraData, ["test": 123])
         }
     }
+
+    func test_attachmentsAreCopiedToSandbox_beforeBeingSent() throws {
```

Comment:
It would be nice to have an E2E Test to test the whole flow 🤔 WDYT? @testableapple @polqf